samples/ProjectGrandmaster.Unity/Assets/AudioSystem.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
```

Comment:
Please ensure that each .cs file has a license header, add the following header to each file:
`// Licensed under the MIT License. See LICENSE in the project root for license information.﻿`

Reply:
FWIW, this license header was recently reduced. There wasn't an ask to update pre-existing files if i understand correctly, but now we only need the following:

// Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.